login.go

```diff
@@ -32,7 +34,8 @@ func (c *LoginController) Authorize(ctx *app.AuthorizeLoginContext) error {
 // Generate runs the authorize action.
 func (c *LoginController) Generate(ctx *app.GenerateLoginContext) error {
 	if !configuration.IsPostgresDeveloperModeEnabled() {
-		return ctx.Unauthorized()
+		jerrors, _ := jsonapi.ConvertErrorFromModelToJSONAPIErrors(models.NewUnauthorizedError("Postgres developer mode not enabled"))
```

Comment:
Constructing a model level error and then constructing an api level error out of it seems strange. I would just construct and jsonapi errors object directly in these cases.